substrate/frame/session/src/mock.rs

```diff
@@ -275,3 +294,21 @@ impl crate::historical::Config for Test {
 	type FullIdentification = u64;
 	type FullIdentificationOf = sp_runtime::traits::ConvertInto;
 }
+
+#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig as pallet_balances::DefaultConfig)]
```

Comment:
Please remove the stuff that is already derived from this line?

Reply:
thx ! I was just mimicking the other pallets and didnt know what it was for. Now i know :)